BUILD

```diff
@@ -195,12 +195,14 @@ cc_library(
         "lib/Conversion/TorchToTcp/TcpCustomOp.cpp",
         "lib/Conversion/TorchToTcp/TorchToTcp.cpp",
         "lib/Conversion/TorchToTcp/TorchToTcpCustomOp.cpp",
+        "lib/Conversion/TorchToTcp/TorchToTcpCustomOpConversionHelper.cpp",
```

Comment:
Prefer using the existing `lib/Conversion/TorchToTcp/Utils.cpp` instead?

Reply:
One alternative is to create a `TorchToTcpCustomOpUtils.h` and `.cpp`, to hold those that are specific to custom op cases. Wdyt?

Reply:
Given that `TorchToTcpCustomOp` lib  is not factored as a separate conversion target (i.e. it reuses `TorchToTcp` [target](https://github.com/cruise-automation/mlir-tcp/blob/main/BUILD#L188)), and we already have some utilities in `Utils.cpp` that are shared by both (e.g. `addPatternIfOpInConvertTorchOpsSet`), it might make sense to keep them colocated. There are organizational benefits in doing so, like making future refactorings for code reuse easier.

Reply:
Interesting... In a previous life, we preferred NOT having files like "utils" etc. because they would become a general catch-all of eventually unrelated things and preferred more standalone headers (ideally a header per class decl.) to make dependencies in the C++ file much more granular.

I have no strong preferences here. @sjain-stanford , did you mean to just put the implementation (cpp file) into `Utils.cpp` or also the helper class into `Utils.h`?

Reply:
I'm going by patterns from the `llvm-project`/`torch-mlir` codebases (which we try to mimic in TCP). See some examples below:
- https://github.com/llvm/torch-mlir/blob/main/lib/Conversion/Utils/Utils.cpp
- https://github.com/llvm/llvm-project/blob/main/mlir/lib/Dialect/SCF/Utils/Utils.cpp
- https://github.com/llvm/llvm-project/blob/main/mlir/lib/Dialect/Tosa/Utils/ConversionUtils.cpp

> did you mean to just put the implementation (cpp file) into Utils.cpp or also the helper class into Utils.h?
Let's move the class interface to `Utils.h` as well (this is already included for use by the conversion patterns).